_api-reference/cat/cat-allocation.md

```diff
@@ -42,7 +42,7 @@ GET _cat/allocation/<node_name>
 
 All cat allocation URL parameters are optional.
 
-In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cat/index), you can specify the following parameters:
+In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-reference/cat/index), you can specify the following parameters:
```

Comment:
[Compact and aligned text (CAT) API]({{site.url}}{{site.baseurl}}/api-reference/cat/index)